keyboards/kingly_keys/ave/ortho/rules.mk

```diff
@@ -0,0 +1 @@
+
```

Comment:
```suggestion
```

Reply:
I’ll fix these when I get back to my pc. It’s not letting me apply these to commit